### PR TITLE
Clean-up "convenience install script" include

### PR DESCRIPTION
**Pull request by thaJeztah**
Follow-up to https://github.com/docker/docker.github.io/pull/12647 (first commit is from that PR, so opening as draft)

Current version on docs.docker.com:

Screenshot 2021-04-23 at 15 36 40

And with this PR:

Screenshot 2021-04-23 at 15 49 22

### CODE REVIEW

**Review by benmccann — DISMISSED** [review]

**Comment by thaJeztah** [comment]
@usha-mandya @AkihiroSuda ptal

File: _includes/install-script.md [hunk 1]
```diff
@@ -2,54 +2,84 @@
...
-testing version, use [test.docker.com](https://test.docker.com/) instead. In
-each of the commands below, replace each occurrence of `get` with `test`.
-
-> **Warning**:
->
-Always examine scripts downloaded from the internet before
-> running them locally.
+Docker provides a convenience script at [get.docker.com](https://get.docker.com/)
+to install Docker into development environments quickly and non-interactively.
+The convenience script is not recommended for production environments, but can be
```
Comment (AkihiroSuda) [inline _includes/install-script.md:7]: Can we have a link for the "recommended" way

File: _includes/install-script.md [hunk 2]
```diff
@@ -2,54 +2,84 @@
...
 
+- The script requires `root` or `sudo` privileges to run.
+- The script attempts to detect your Linux distribution and version and
+  configure your package management system for you, and does not allow you to
+  customize most installation parameters.
+- The script installs dependencies and recommendations without asking for
+  confirmation. This may install a large number of packages, depending on the
+  current configuration of your host machine.
+- By default, the script installs the latest stable release of Docker, containerd,
+  and RunC. When using this script to provision a machine, this may result in
```
Comment (AkihiroSuda) [inline _includes/install-script.md:25]: RunC -> runc?

Comment (thaJeztah) [reply _includes/install-script.md:25]: fixed

File: _includes/install-script.md (hunk 1 above)
Comment (thaJeztah) [reply _includes/install-script.md:7]: Added:

```markdown
Also refer to the [install using the repository](#install-using-the-repository)
steps to learn about installation steps to install using the package repository.
```

**Comment by thaJeztah** [comment]
@AkihiroSuda rebased, and made some changes (see comments above); PTAL

**Review by usha-mandya — APPROVED** [review]
LGTM

**Pull request merged**